src/NexusMods.App.UI/Controls/Alerts/Alert.axaml.cs

```diff
@@ -0,0 +1,259 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Markup.Xaml;
+using NexusMods.Icons;
+using ReactiveUI;
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Interactivity;
+
+namespace NexusMods.App.UI.Controls.Alerts;
+
+public class Alert : ContentControl
+{
+    public enum SeverityOptions
+    {
+        None,
+        Info,
+        Success,
+        Warning,
+        Error
+    }
+
+    /// <summary>
+    /// Defines the Title property of the <see cref="Alert"/>. Defaults to "Default Title".
+    /// </summary>
+    public static readonly StyledProperty<string?> TitleProperty = AvaloniaProperty.Register<Alert, string?>(nameof(Title), defaultValue: "Default Title");
+
+    /// <summary>
+    /// Defines the Body property of the <see cref="Alert"/>. Defaults to "Default Body".
+    /// </summary>
+    public static readonly StyledProperty<string?> BodyProperty = AvaloniaProperty.Register<Alert, string?>(nameof(Body), defaultValue: "Default Body");
```

Comment:
I guess you are using these for preview? I would usually have empty values as the default otherwise.